cellprofiler/modules/correctilluminationcalculate.py

```diff
@@ -206,209 +226,222 @@ def create_settings(self):
-            Enter a name that will allow the dilated image to be selected later in the pipeline.''')
+            "Name the dilated image", "IllumBlueDilated", doc="""\
+*(Used only if the dilated image is to be retained for later use in the pipeline)*
+Enter a name that will allow the dilated image to be selected later in
+the pipeline.""")
 
         self.automatic_splines = cps.Binary(
-                "Automatically calculate spline parameters?", True, doc="""
-            <i>(Used only if %(SM_SPLINES)s are selected for the smoothing method)</i><br>
-            Select <i>%(YES)s</i> to automatically calculate
-            the parameters for spline fitting.
-            <p>Select <i>%(NO)s</i> to specify the background mode, background threshold, scale,
-            maximum number of iterations and convergence.</p>""" % globals())
+"Automatically calculate spline parameters?", True, doc="""\    
+*(Used only if %(SM_SPLINES)s are selected for the smoothing method)*
+Select *%(YES)s* to automatically calculate the parameters for spline
+fitting.
+
+Select *%(NO)s* to specify the background mode, background threshold,
+scale, maximum number of iterations and convergence.
+""" % globals())
 
         self.spline_bg_mode = cps.Choice(
-                "Background mode",
-                [MODE_AUTO, MODE_DARK, MODE_BRIGHT, MODE_GRAY], doc="""
-            <i>(Used only if %(SM_SPLINES)s are selected for the smoothing method and
-            spline parameters are not calculated automatically)</i><br>
-            This setting determines which pixels are background
-            and which are foreground.<br>
-            <ul><li><i>%(MODE_AUTO)s</i>: Determine the mode from the image.
-            This will set the mode to %(MODE_DARK)s if most of the pixels are
-            dark, %(MODE_BRIGHT)s if most of the pixels are bright and
-            %(MODE_GRAY)s if there are relatively few dark and light pixels
-            relative to the number of mid-level pixels</li>
-            <li><i>%(MODE_DARK)s</i>: Fit the spline to the darkest pixels
-            in the image, excluding brighter pixels from consideration.
-            This may be appropriate for a fluorescent image.
-            </li>
-            <li><i>%(MODE_BRIGHT)s</i>: Fit the spline to the lightest pixels
-            in the image, excluding the darker pixels. This may be appropriate
-            for a histologically stained image.</li>
-            <li><i>%(MODE_GRAY)s</i>: Fit the spline to mid-range pixels,
-            excluding both dark and light pixels. This may be appropriate
-            for a brightfield image where the objects of interest have
-            light and dark features.</li></ul>""" % globals())
+            "Background mode",
+            [MODE_AUTO, MODE_DARK, MODE_BRIGHT, MODE_GRAY], doc="""\              
+*(Used only if %(SM_SPLINES)s are selected for the smoothing method
+and spline parameters are not calculated automatically)*
```

Comment:
Add a newline between `*(Used only if...)*` and the next line. This will insert a line break in the rendered documentation. Otherwise, the text appears on the same line.